resources/basics.py

```diff
@@ -54,18 +54,46 @@
 }
 
 
+def get_release_file(file_path: str, version: str = CURRENT_RELEASE) -> str:
+    """
+    Tries to get the desired file from the corresponding release version on the google cloud.
+    If the file is not found for the desired release version, falls back on previous versions.
+
+    :param  str file_path: Desired file path, with {0} as placeholder(s) for the version number
+    :param str version: Desired file version
+    :return: Path for closest version of the file available
+    """
+    if hl.hadoop_exists(file_path.format(version)):
```

Comment:
Should we be checking for the _SUCCESS file?

Reply:
I wrote this function the other day to do just this, perhaps this could be useful here:

```
def check_hail_file(fname):
    return hl.hadoop_exists(f'{fname}/_SUCCESS')
```